run-tox-env.yml

```diff
@@ -87,6 +87,14 @@ jobs:
           - bash: docker run -v $PWD:/project -i --name ${{ variables.docker_name }} -d ${{ variables.docker_image }} /bin/bash
             displayName: Start up Docker container
 
+        - ${{ if parameters.cache_dir }}:
+          - bash: mkdir ${{ parameters.cache_dir }} && touch '${{ parameters.cache_dir }}/azure-template-empty-file'
```

Comment:
This is working, but on windows it doesn't like the /, does anyone know a way to make these commands platform independent?